torch_frame/data/dataset.py

```diff
@@ -111,6 +112,12 @@ class DataFrameToTensorFrameConverter:
             maps sentences into :class:`torch.nn.Embeddings` and
             :obj:`batch_size` that specifies the mini-batch size for
             :obj:`text_embedder`. (default: :obj:`None`)
+        text_tokenizer_cfg
+            (:class:`torch_frame.config.TextTokenizerConfig`, optional):
+            A text tokenizer config specifying :obj:`text_tokenizer` that
+            maps sentences into tensor of tokens and
+            :obj:`batch_size` that specifies the mini-batch size for
+            :obj:`text_tokenizer`. (default: :obj:`None`)
```

Comment:
I found some incoherence of text tokenizer definition. I think this definition makes sense. Can we make this consistent? See my previous comments as well.